### PR TITLE
Responsive menus

### PR DESCRIPTION
**Pull request by Sesquipedalian**
Shows descriptive labels for the mobile menu links and tidies up the presentation of the mobile menus themselves.
Screen Shot 2022-01-26 at 9 25 35 PM
Screen Shot 2022-01-26 at 9 26 58 PM

### CODE REVIEW

**Comment by BrickOzp** [comment]
Something happened to the menues
image

**Comment by DiegoAndresCortes** [comment]
I see we are fighting our nemesis again
https://www.youtube.com/watch?v=c7BVtGnlxT8

**Comment by Sesquipedalian** [comment]
> Something happened to the menues

Should be fixed in d2731f8.

> https://www.youtube.com/watch?v=c7BVtGnlxT8

Poor kid! 😳😆 Just wanna hug him.

File: Themes/default/css/responsive.css [hunk 1]
```diff
@@ -562,11 +555,56 @@
...
 		left: 0;
+		box-shadow: none;
+	}
+	/* 3rd level menu tests */
+	.dropmenu li ul ul {
+		margin: 0 !important;
+	}
+}
+
+@media (min-width: 480px) and (max-width: 560px) {
```
Comment (DiegoAndresCortes) [inline Themes/default/css/responsive.css]: Should be min-width **481**

**Review by DiegoAndresCortes — CHANGES_REQUESTED** [review]

File: Themes/default/css/responsive.css (hunk 1 above)
Comment (Sesquipedalian) [reply Themes/default/css/responsive.css]: That's been wrong for a long time...

Fixed in 697406e.

**Review by DiegoAndresCortes — APPROVED** [review]

**Pull request merged**